File: PHP Word Examples/ConvertApiDemo/convertToVectorgraphInRequest.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\ConvertApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ConvertApi($configuration);

$inputFile = "D:/data/ConvertToVectorgraphInRequest.docx";
$image_type = "wmf"; //Available type: wmf, svg
$password = null;
$start_page = null;
$page_count = null;
 $result = $apiInstance->convertToVectorgraphInRequest(
     $inputFile,
     $image_type,
     $password,
     $start_page,
     $page_count
 );

File: Python Word Examples/ConvertApiDemo/convertToVectorgraphInRequest.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.convert_api.ConvertApi(configuration)

file_name = 'D:/input/ConvertToVectorgraphInRequest.docx'
image_type = 'wmf' #Available values : wmf, svg
result = api.convert_to_vectorgraph_in_request(file_name, image_type)